test/e2e/flask/solana/solana-networks.spec.ts

```diff
@@ -0,0 +1,60 @@
+import { Suite } from 'mocha';
+import { switchToNetworkFlow } from '../../page-objects/flows/network.flow';
+import HeaderNavbar from '../../page-objects/pages/header-navbar';
+import AccountListPage from '../../page-objects/pages/account-list-page';
+import { withSolanaAccountSnap } from './common-solana';
+
+describe('Solana network', function (this: Suite) {
+  it.skip('keeps the network selector enabled when the Solana network is selected', async function () {
+    await withSolanaAccountSnap(
+      { title: this.test?.fullTitle() },
+      async (driver) => {
+        const headerNavbar = new HeaderNavbar(driver);
+        await headerNavbar.check_pageIsLoaded();
+        await headerNavbar.check_accountLabel('Solana 1');
+        await headerNavbar.check_currentSelectedNetwork('Solana Mainnet');
+        await headerNavbar.check_ifNetworkPickerClickable(true);
+        await headerNavbar.openAccountMenu();
+        const accountMenu = new AccountListPage(driver);
+        await accountMenu.switchToAccount('Account 1');
+        await headerNavbar.check_currentSelectedNetwork('Localhost 8545');
+        await headerNavbar.check_ifNetworkPickerClickable(true);
+      },
+    );
+  });
+
+  it('deletes previously selected EVM network when Solana network is selected', async function () {
```

Comment:
Feels like the current wording is misleading, like if "selecting the Solana network will delete the previously selected EVM network" 😅 WDYT about this?

```suggestion
  it('can delete the previously selected EVM network when Solana network is selected', async function () {
```